crates/bevy_pbr/src/decal/clustered.rs

```diff
@@ -367,3 +367,16 @@ fn upload_decals(
 
     decals_buffer.write_buffer(&render_device, &render_queue);
 }
+
+/// Returns true if clustered decals are usable on the current platform or false
+/// otherwise.
```

Comment:
```suggestion
/// otherwise.
///
/// Clustered decals are currently disabled on MacOS and iOS due to insufficient texture bindings and limited bindless support in wgpu.
```

Putting this rationale in the docs helps users who are trying to piece this together without opening up the source code.